### PR TITLE
Install pgbouncer to Carrenza postgresql-primary machines

### PR DESCRIPTION
**Pull request by barrucadu**
[Trello card](https://trello.com/c/BosbZ9n9/336-investigate-postgres-connection-pooling)

### CODE REVIEW

File: modules/govuk/manifests/node/s_postgresql_primary.pp [hunk 1]
```diff
@@ -21,6 +21,8 @@
     slave_password => $standby_password,
   }
 
+  include govuk::pgbouncer
```
Comment (boffbowsh) [inline modules/govuk/manifests/node/s_postgresql_primary.pp]: Think this needs to be `::govuk::pgbouncer`

Comment (barrucadu) [reply modules/govuk/manifests/node/s_postgresql_primary.pp]: `s_postgresql_base` doesn't have the leading `::`: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_postgresql_base.pp

**Comment by barrucadu** [comment]
Are there any load balancers that connections will go through, or is pgbouncer allowing the `backend_ip_range` (`"%{hiera('environment_ip_prefix')}.3.0/24"`) enough?

**Comment by boffbowsh** [comment]
That should be enough.

**Comment by barrucadu** [comment]
How about firewall rules to open ports?

**Comment by boffbowsh** [comment]
I *think* we only have firewalls between networks, and this is all within the same network for now.

**Pull request merged**